cwms_radar_api/src/main/java/cwms/radar/data/dto/TimeSeries.java

```diff
@@ -38,6 +41,7 @@
 @XmlAccessorType(XmlAccessType.FIELD)
 @XmlAccessorOrder(XmlAccessOrder.ALPHABETICAL)
 @JsonPropertyOrder(alphabetic = true)
+@JsonNaming(PropertyNamingStrategies.KebabCaseStrategy.class)
```

Comment:
I assume this changed how the JSON is generated?  There is a unit test TimeSeriesTest that does some serialization tests and currently checks some of the xml fields.  This change seems like it would be easy to test in that class

Reply:
Gah, thanks. And yes, I think I updated the json but need to verify again and make sure that's still required. There was some oddness with how the names were generated in the openapi schema vs when the formatters actually rendered it.